package exceptiondemo2;

public class StackOverflowErrorDemoNew {

	public static void main(String[] args) {
		
		new DemoErrorB();

	}

}
